### PR TITLE
Meteoblue intergration

### LINKED ISSUE #3: Link to meteoblue.com for the current mountain range

### PR DESCRIPTION
**Pull request by gabriel-v**
Closes #3 

I couldn't figure out how to properly display the `meteoblue-logo.svg` image over the link, so I used the `.png` variant instead.

Added a gulp task for copying over the `media` folder to `build`. I know this is may be done elsewhere, but it's nice to have gulp watch that folder, too.

### CODE REVIEW

File: src/data.coffee [hunk 1]
```diff
@@ -14,18 +14,18 @@ data = module.exports = {}
...
-  ciucas:     {bbox: [25.84, 45.43, 26.05, 45.56], title: "Ciucaș"}
-  crai:       {bbox: [25.15, 45.49, 25.30, 45.58], title: "Piatra Craiului"}
-  fagaras:    {bbox: [24.30, 45.47, 24.89, 45.73], title: "Făgăraș"}
-  iezer:      {bbox: [24.85, 45.38, 25.10, 45.55], title: "Iezer"}
-  macin:      {bbox: [28.13, 45.07, 28.42, 45.29], title: "Măcin"}
-  nerei:      {bbox: [21.70, 44.80, 22.00, 45.03], title: "Cheile Nerei"}
-  parang:     {bbox: [23.43, 45.31, 23.82, 45.41], title: "Parâng"}
-  piatramare: {bbox: [25.59, 45.48, 25.75, 45.60], title: "Piatra Mare"}
-  retezat:    {bbox: [22.72, 45.29, 23.00, 45.42], title: "Retezat"}
+  bucegi:     {bbox: [25.36, 45.33, 25.58, 45.52], title: "Bucegi", meteoLink: "https://www.meteoblue.com/en/weather/forecast/week/mun%C5%A3ii-bucegi_romania_683598"}
```
Comment (mgax) [inline src/data.coffee]: How about we only specify the slug (`mun%C5%A3ii-bucegi_romania_683598` in this case) and build the full URL when we display the link?

**Comment by mgax** [comment]
Looking good! One more nit to pick: can we have a single task `assets`, in place of `libs` and `media`? They essentially do the same thing: copy static 3rd party content.

File: src/data.coffee (hunk 1 above)
Comment (gabriel-v) [inline src/data.coffee]: Sure, that'll work even if we have something like  `https://www.meteoblue.com/en/weather/forecast/week/44.946N21.863E`. Will do

**Comment by gabriel-v** [comment]
`libs` gets static 3rd party content from CDN, one-time deal. `media` runs a deep copy of our own files. I think `media` should be watched in case someone wants to fiddle with some media file (like some image) but this doesn't apply for `libs`.

**Comment by mgax** [comment]
That's more complexity than we need at this point - there's no media files to fiddle with.

**Comment by mgax** [comment]
:cake: ^_^

**Pull request merged**